core/overlay/conversion.go

```diff
@@ -319,7 +370,7 @@ func OverlayVerkleTransition(statedb *state.StateDB, root common.Hash, maxMovedC
 				// processing the storage for that account where we left off.
 				// If the entire storage was processed, then the iterator was
 				// created in vain, but it's ok as this will not happen often.
```

Comment:
Changes from here until the end of the file are just to always migrate "full accounts" -- just to discard any half-migrated situations.